### ISSUE TITLE
userAction.click doesn't work VideoJS v7.17.0

### ISSUE THREAD

**Issue by DJ-PD** [videojs/video.js#7577]
Using VideoJS as an audio player with use of an mp3 live stream source via the <audio> element.
Trying to inactivate the player window play/pause toggle function with use of the new function "userAction.click"
Shouldn't I be able to set this to false via the player options "userActions" as below example?
Tried several different ways but I can't get it to work.
`                            
options = {	                       
controls: true,                         
autoplay: false,                          
muted: false,														
**userActions: { click: false },**                         
preload: 'none',                          
poster: [ 'img/pd-stream1920x1080.jpg' ],	                        
techOrder: [ 'chromecast', 'html5' ],`

**Comment by mister-ben** [comment]
That should work, have you a live example where it is not?

**Comment by DJ-PD** [comment]
Hmm, interesting, I tried again both on local as well on the public server and everything works without the userAction false function. I don't want to share the link public here but if you mail me at neccen@gmail.com I will share it with you. We will go live over the Christmas 24 and 25th.

**Comment by DJ-PD** [comment]
Made a new audio test and regardless if I use videoJS version 7.16, 7.17 or 7.18 via a mp3 live stream or just a mp3 file with or without the option `userActions: { click: false }` it seems like the poster option `poster: [ 'img/pd-stream1920x1080.jpg' ]` takes over, see below print screen...
![image](https://user-images.githubusercontent.com/46725109/147291475-9628abb3-5a13-4590-81a4-f784d9d993cc.png)

**Comment by mister-ben** [comment]
Are you using an `<audio>` element? `userActions.click` affects clicks on the playback el, but you're clicking on the `posterImage` component instead.

You can either use a `<video>` instead (which can play audio), or remove the click handler from the poster with `player.posterImage.off('click')`

**Comment by DJ-PD** [comment]
Yes, I use the `<audio>` element. Made a new test via the `<video>` element as you mentioned and that works good in combination of the option `userActions: { click: false }` even with a poster. Downside is that I lose the all time activated control bar which I (users) see as a benefit when just playing audio. Can I trigg the control bar to be activated all the time even with use of the `<video>` element?
I can also see that the control bar icons toggle the pause/play function with use of a mobile device (Android tested right now) which is extremely enjoying i.e tap on the "full screen" or "chromecast" icon pause the audio where the user do need to tap again on the same icon to activate the icon function. See below print screen.
![image](https://user-images.githubusercontent.com/46725109/147390938-f7693110-740b-45a8-b939-be24deeab7ae.png)

**Comment by mister-ben** [comment]
You'll need this combination for now:

* `<audio>`, or `<video-js>` with a `vjs-audio` class. This will keep the controls in view
* `.vjs-audio.vjs-has-started .vjs-poster { pointer-events: none; }` to inhibit clicks on the poster during playback

We'll need to think about whether `userActions.click` should be applied to the poster as well as the tech for audio players.
